airbyte-protocol/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -224,8 +224,7 @@ definitions:
     required:
       - name
       - json_schema
-      # todo (cgardens) - make required once sources are migrated
-      # - supported_sync_modes
+      - supported_sync_modes
```

Comment:
I guess we've migrated!  Can you confirm that there are no old source that don't provide this property?

Reply:
I'm not 💯 of the Singer compatibility...

Reply:
> Can you confirm that there are no old source that don't provide this property?
We can double check that existing GA/beta connectors SAT is passing - a test to check the presence of this property was shipped a week ago #14800

Reply:
I'm going to add @jdpgrailsdev as well to confirm, but I think this is 👍